HarClient.Tests/Mocks/MockServerHandler.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Net;
+using System.Net.Http;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace HarClient.Tests.Mocks
```

Comment:
All the namespaces should match the source project, that is JBlam.HarClient.Tests(.*)